src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Color getRandomColor() {
+        int colorLength = Color.values().length;
+        Random colorBall = new Random();
```

Comment:
```suggestion
        Random random = new Random();
```

 змінну random треба винести на рівень класу, що б не створювати кожен раз екземпляр цього класу при вилику методу  getRandomColor()